modules/hibernating-cluster-resume.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/hibernating-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hibernating-cluster-resume_{context}"]
+= Resuming a hibernated cluster
+
+You can resume a hibernated cluster within 90 days. You might have to approve certificate signing requests (CSRs) for the nodes to become ready.
+
+It can take around 45 minutes for the cluster to resume, depending on the size of your cluster.
+
+.Prerequisites
+
+* You hibernated your cluster less than 90 days ago.
++
+// TODO: What if they try to resume later than 90 days? What are the consequences?
+
+* You have access to the cluster as a user with the `cluster-admin` role.
++
+// TODO: Confirm must be a cluster-admin
+
+.Procedure
+
+. Within 90 days of cluster hibernation, resume the cluster virtual machines:
++
+Use the tools native to your cluster's cloud environment to resume the cluster's virtual machines.
+
+. Wait about 5 minutes, depending on the number of nodes in your cluster.
+
+. TODO: Check node status. Sync w/ Vadim on possible script?
```

Comment:
@vrutkovs in the google doc you mentioned a possible script instead of checking the node status. Do you have an idea of what we should do here?

Reply:
The script would be `oc --request-timeout=5s wait --for=condition=Progressing clusteroperator/kube-apiserver --timeout=300s` which waits for kube-apiserver operator to be progressing again. But we found out that its unreliable - see https://issues.redhat.com/browse/API-1614 - so for now we have to do with "wait for 5 minutes and approve CSRs as they appear"

Reply:
Okay thanks, I'll keep it as waiting and then making sure there's a CSR for each NotReady node